src/instructlab/cli/process/process.py

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Standard
+import logging
+
+# Third Party
+import click
+
+# First Party
+from instructlab import clickext
+
+logger = logging.getLogger(__name__)
+
+
+@click.group(
+    cls=clickext.LazyEntryPointGroup,
+    ep_group="instructlab.command.process",
+)
+@click.pass_context
+def process(ctx):
+    """Command group for all process-related command calls"""
+    ctx.obj = ctx.parent.obj
```

Comment:
as discussed in the dev doc, we cannot be storing process information in the click context due to upcoming work with the REST API

Reply:
what I try to do is based on the currently resource (what we have now).